provider/datagen/src/driver.rs

```diff
@@ -370,23 +385,32 @@ impl DatagenDriver {
                         .iter()
                         .try_for_each(|(locale, (payload, _duration))| {
                             let mut iter = fallbacker_with_config.fallback_for(locale.clone());
+                            let mut first = true;
```

Comment:
The effect of `first` is that it prevents non-base locales such as `tlh-001` from being retained. I'm not sure if this is the behavior that we desire; it only comes up when an unsupported locale is requested and that locale is not a base language. I lean toward removing the check on `first`, which would cause `tlh-001` to be retained.

Reply:
I think it's fine to remove `first`, but we should probably print a warning alerting the user that the provider locale is not supported.

Reply:
Yep. Today I'm thinking that we probably should not include _either_ `tlh-001` _or_ `arc` since there is no CLDR data for them, and therefore they are not really "supported". I'll work on that tweak.